README.md

```diff
@@ -20,9 +20,9 @@ files are often a critical aspect of the file, holding ACLs, capabilities, etc.
 While FreeBSD filesystem do support `extattr`, this feature has not made its
 way into their `mtree`.
 
-This implementation of mtree supports an additional "keyword" of `xattr`. If
-you include this keyword, then the FreeBSD `mtree` will fail as it is an
-unknown keyword to that implementation.
+This implementation of mtree supports a few non-upstream "keyword"s. These are:
+`xattr` and `tar_time`. If you include these keywords, then the FreeBSD `mtree`
+will fail as they are unknown keywords to that implementation.
```

Comment:
I was thinking it would be useful to mention more strict compatibility by using the `-bsd-keywords` flag here. Maybe a new paragraph.